pkg/driver/nydus/nydus.go

```diff
@@ -30,12 +35,58 @@ import (
 	"github.com/goharbor/acceleration-service/pkg/utils"
 )
 
+type chunkDictInfo struct {
+	BootstrapPath string
+	Blobs         map[string]ocispec.Descriptor
```

Comment:
Why do we need `Blobs` given it's not really used?

Reply:
Fixed, thanks.